providers/linux/os.go

```diff
@@ -205,7 +205,7 @@ func findDistribRelease(baseDir string) (*types.OSInfo, error) {
 		}
 
 		info, err := os.Lstat(path)
-		if err != nil || !info.Mode().IsRegular() || info.Size() == 0 {
+		if err != nil || info.Size() == 0 {
```

Comment:
The glob matches anything like `/etc/*-release`. Due to us removing the check we hope that this path is always a file (or symlink to a file). Otherwise the read operation following will fail.

I presume it's unlikely that a path matching this pattern is not a file.

Reply:
I agree.  I think it's unlikely that the path will be anything other than a file or a symlink to a file.